cluster/manifests/wiz/connector/clusterrole.yaml

```diff
@@ -0,0 +1,39 @@
+{{ if eq .Cluster.ConfigItems.wiz_enable_runtime_monitoring_daemonset "true"}}
+# ---
+# # We are using ClusterRole readonly created by default in the cluster instead of creating new one provided by wiz
+# # Source: wiz-kubernetes-integration/charts/wiz-kubernetes-connector/templates/service-account-cluster-reader.yaml
+# apiVersion: rbac.authorization.k8s.io/v1
+# kind: ClusterRole
+# metadata:
+#   name: wiz-cluster-reader
+#   labels:
+#     helm.sh/chart: wiz-kubernetes-connector-3.1.1
+#     app.kubernetes.io/name: wiz-kubernetes-connector
+#     app.kubernetes.io/instance: wiz-connector
+#     app.kubernetes.io/version: "2.5"
+#     app.kubernetes.io/managed-by: Helm
+# rules:
+#   - apiGroups: ["*"]
+#     resources: ["*"]
+#     verbs: ["get", "list", "watch"]
+---
+# Source: wiz-kubernetes-integration/charts/wiz-kubernetes-connector/templates/service-account-cluster-reader.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: wiz-cluster-reader
+  labels:
+    helm.sh/chart: wiz-kubernetes-connector-3.1.1
+    app.kubernetes.io/name: wiz-kubernetes-connector
+    app.kubernetes.io/instance: wiz-connector
+    app.kubernetes.io/version: "2.5"
+    app.kubernetes.io/managed-by: Helm
```

Comment:
Are these labels important for the Wiz deployment? I think we might need to remove the ones related to `helm`, also we need to provide the Zalando environment labels like `application: foo` and `component: bar`.

Reply:
I see a similar concern was raised by Martin in another manifest, so let's fix the labels for all manifests 🙂 
1. Drop the helm related labels.
2. Add `application` and `component` labels.